### ISSUE TITLE
bass boost in the app

### ISSUE THREAD

**Issue by Satbeer74** [naman14/Timber#431]
Sir the bass boost option in the app is not operating while turning on the equilizer.

**Comment by mikhoul** [comment]
The equalizer is independent of Timber, Timberr just use the equalizer that is installed in your Android device.

To give an exemple I use this Equalizer and it work fine with Timber: https://play.google.com/store/apps/details?id=com.devdnua.equalizer.free :octocat:

**Comment by Satbeer74** [comment]
But sir if we want to include the equalizer in the app then how we can use it.

**Comment by Satbeer74** [comment]
Yes sir u told this but the matter of fact is that we can access the equilizer section but we cannot adjust the bass boost option.
